docs/UserGuide.md

```diff
@@ -146,6 +186,9 @@ For example, you can charge the third guest `30.35` by running the command below
 
 **:information_source: Notes about the command format:**<br>
```

Comment:
Should we use call out with color-coded backgrounds for information_source, bulb,  triangular_flag_on_post and bangbang to make them more obvious?

Reply:
Yes, the colours have implemented already with the existing code.